tests/manager/manager-ctor-read_preference-003.phpt

```diff
@@ -1,5 +1,5 @@
 --TEST--
-MongoDB\Driver\Manager::__construct(): read preference options of the wrong type are ignored
+MongoDB\Driver\Manager::__construct(): read preference options of the wrong type will not be ignored
```

Comment:
I would leave this test as-is for now. Once all of the read preference options are handled in the PR, we can simply delete it.